src/stream-to-buffer.iced

```diff
@@ -0,0 +1,14 @@
+stream = require('stream')
+
+exports.StreamToBuffer = class StreamToBuffer extends stream.Transform
+
+  constructor : (options) ->
+    @bufs = []
+    super(options)
+
+  _write : (chunk, encoding, cb) ->
+    @bufs.push(chunk)
```

Comment:
Just out of curiosity, any reason not to go ahead and concat things we they're written, rather than all at once at the end?